lib/crawler/api/crawl.rb

```diff
@@ -77,11 +77,6 @@ def interruptible_sleep(period)
         end
       end
 
-      # No errors should be retried by default (see App Search subclass for a version with retries)
-      def retryable_error?(_error)
-        false
-      end
-
```

Comment:
This isn't used anymore so I've removed it. It was used for App Search crawlers.